src/main/java/bio/terra/service/exception/ExternalServiceException.java

```diff
@@ -0,0 +1,17 @@
+package bio.terra.service.exception;
+
+import bio.terra.exception.InternalServerErrorException;
+
+public class ExternalServiceException extends InternalServerErrorException {
```

Comment:
Is your thought here to make one common exception for all external services?
I have separate exceptions defined for FireStore, BigQuery, and GCS right now.
My bias is to be more specific.

Reply:
Yep, I'll make it more specific.